index.html

```diff
@@ -465,6 +465,15 @@ <h4 id="shell-linux">Linux</h4>
     web browser (current versions of Chrome, Firefox or Safari,
     or Internet Explorer version 9 or above).
   </p>
+  <p>
+    You will need an account at <a href="https://github.com/">github.com</a>
+    for parts of the git lesson. Basic Github accounts are free. We encourage
```

Comment:
nits: "git" -> "Git" when referring to the Project and not just the executable.  And "GitHub" -> "GitHub".